### DISCUSSION TITLE
Span type for tool invocations

### DISCUSSION THREAD

**Discussion by jannikmaierhoefer** [langfuse/langfuse#4665] [Ideas]
### Describe the feature or potential improvement

It would be helpful to have a specific Langfuse span type for tool calls.

An alternative would be to add an attribute for function calls, e.g., "TOOL-CALL"  (existing attributes: DEBUG, DEFAULT, WARNING, ERROR).

**Comment by vblagoje** [comment]
@maxdeichmann it would be awesome to leave some references - which release, doc links etc, as I can't find info in the docs about these span traces for tools apart from https://youtu.be/IkSK5Kz-Pt8?si=LxmAtw9LshaYa1lq

**Reply by maxdeichmann** [reply]
@vblagoje sorry for the confusion here. I mixed something up. This is not implemented yet, hence re-opening this.

**Comment by jennyma** [comment]
@jannikmaierhoefer Have you seen any progress here?

**Comment by vblagoje** [comment]
Yes we implemented tool invocation and other span types. See https://github.com/deepset-ai/haystack-core-integrations/blob/main/integrations/langfuse/CHANGELOG.md#integrationslangfuse-v320---2025-11-07 and https://github.com/deepset-ai/haystack-core-integrations/blob/main/integrations/langfuse/CHANGELOG.md#-features-1
